### ISSUE TITLE
map plugin not working

### ISSUE THREAD

**Issue by sahilshekhawat** [tdt/rdf2html#6]
I tried the given examples and map doesn't seems to work.

**Comment by coreation** [comment]
Hi @sahilshekhawat, can you provide the data and configuration that you're using?

**Comment by pietercolpaert** [comment]
Indeed! In examples/index2.html, the map doesn't show up... To be investigated.
